webpack.config.js

```diff
@@ -170,11 +171,13 @@ async function fetchAppConfigAndEnvironmentVars() {
 
   const { shortlink_domain, thumbnail_server } = hubsConfigs.general;
 
+  const localIp = await internalIp.v4();
+
   process.env.RETICULUM_SERVER = host;
   process.env.SHORTLINK_DOMAIN = shortlink_domain;
-  process.env.CORS_PROXY_SERVER = "localhost:8080/cors-proxy";
+  process.env.CORS_PROXY_SERVER = `${localIp}:8080/cors-proxy`;
```

Comment:
May want to include or fallback to `localhost` in case `internalIp.v4()` returns `undefined` (e.g. if `ip` is not installed https://github.com/sindresorhus/internal-ip#usage)